mfthomps/CyberCIEGE-port-to-Unity3d
# Refactor asset bundles (resolves #362, #363)

To test:
* run Introduction scenario in the editor - should be same as on `develop`
* try to build standalone - should build and run without the need to copy asset bundle into `..._Data`

(closes #362, #363)
